src/views/virtualmachines/list/components/VirtualMachineRow/VirtualMachineRow.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { RowProps } from '@openshift-console/dynamic-plugin-sdk';
+
+import { printableVMStatus } from '../../../utils';
+
+import VirtualMachineRowLayout from './VirtualMachineRowLayout';
+import VirtualMachineRunningRow from './VirtualMachineRunningRow';
+
+const NO_DATA_DASH = '-';
```

Comment:
if you have a utils file this can exported there :)

Reply:
+1 changed thanks :)